README.md

```diff
@@ -7,3 +7,26 @@ A repository for team discussion, syncing, and meeting notes.
 Developer Meetings take place on [zoom](https://zoom.us/my/jovyan?pwd=c0JZTHlNdS9Sek9vdzR3aTJ4SzFTQT09), every other Tuesday ([calendar](https://jupyter.org/community#calendar)) at 8:15AM Pacific Time ([your time](https://www.thetimezoneconverter.com/?t=8%3A30%20am&tz=San%20Francisco)).
 
 Minutes are taken at [Hackmd.io](https://hackmd.io/5XWHyOoLTRqyXzEHsVmxXg).
+
+## Jupyter Widgets Council
+
+### Current team members
+
+This page lists (alphabetically) the officially named Jupyter Widgets Team.
+
+David Brochart
+Sylvain Corlay
+Matt Craig
+Itay Dafna
+Vidar Tonaas Fauske
+Jason Grout
+Paul Ivanov
+Mariana Meireles
+Jeremy Tuloup
+Martin Renou
```

Comment:
Maybe using a list so the names render better?

![image](https://github.com/jupyter-widgets/team-compass/assets/591645/4d580bcd-2bb2-4fc1-9d7e-187ca97bc8b4)

```suggestion
- David Brochart
- Sylvain Corlay
- Matt Craig
- Itay Dafna
- Vidar Tonaas Fauske
- Jason Grout
- Paul Ivanov
- Mariana Meireles
- Jeremy Tuloup
- Martin Renou
```